### ISSUE TITLE
GIMP keeps asking to update after I uninstall its manual

### ISSUE THREAD

**Issue by hyuri** [pop-os/shop#318]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="21.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.10"
VERSION_ID="21.10"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=impish
UBUNTU_CODENAME=impish
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
GIMP 2.10.30

**Issue/Bug Description:**
Title says it all:
If I uninstall GIMP's manual (flatpak uninstall org.gimp.GIMP.Manual), Pop!_Shop keeps nagging me to update GIMP even though I already have the latest version.

**Steps to reproduce (if you know):**
(have latest version of GIMP flatpak installed)
1. Open a terminal;
2. Remove GIMP's manual using the command "flatpak uninstall org.gimp.GIMP.Manual";
3. Wait until Pop!_Shop checks for updates, or open Pop!_Shop and wait for it to check for updates. Once done, the latest version of GIMP (that you already have installed) will show up as an update.

I've clicked on "Update" once and then uninstalled the manual again, and the issue comes back.

**Expected behavior:**
Intentionally uninstalling a program's component -- in this case, one that I don't use, so wastes space on my drive -- should not trigger an update to the given program.

It would be nice if the next *actual* update to the given program would refrain from installing the component you have intentionally removed in your current installation. I understand that this might be complex to implement, but at least a program's manual should be a separate component that you opt-in to install, since not everyone needs it, and most people will look at the online version anyways.

**Other Notes:**

**Comment by isantop** [comment]
Do you have both the Flatpak and Apt versions installed?

**Comment by hyuri** [comment]
No, only the Flatpak.

**Comment by jacobgkau** [comment]
I tested this today on Pop!_Shop version 7.3.0. Installing the GIMP flatpak did not install `org.gimp.GIMP.Manual`. After installing that package manually with `flatpak install org.gimp.GIMP.Manual`, then removing it with `org.gimp.GIMP.Manual` (after first opening both the GIMP Help Browser and the Pop!_Shop), the Pop!_Shop is not reporting any updates available.

![Screenshot from 2023-10-09 12-21-59](https://github.com/pop-os/shop/assets/7199422/565df4ae-881b-4343-939d-37483f04c5df)

It seems that whatever was causing this has been fixed.
